tox.ini

```
@@ -142,6 +142,7 @@ envlist =
     sanic_contrib-py{36,37,38}-sanic
     sqlalchemy_contrib-py{27,35,36,37,38}-sqlalchemy{10,11,12,13,}-psycopg228-mysqlconnector
     sqlite3_contrib-{py27,py35,py36,py37,py38}-sqlite3
+    starlette_contrib-{py36,py37}-starlette{138}
```

The empty element in the list will make sure that we check latest starlette.

```suggestion
    starlette_contrib-{py36,py37,38}-starlette{138,}
```